### PR TITLE
made a readme

### PR DESCRIPTION
**Pull request by Crystal15118**
Mostly just copied over and formatted the !!seedqueue command in javacord

### CODE REVIEW

**Comment by Crystal15118** [comment]
i have no idea what [Merge branch 'KingContaria:main' into Crystal15118-pull-requests](https://github.com/KingContaria/seedqueue/pull/28/commits/3278e1cade2ccda8388efe1e5613d338093c6a77) did so if anyone knows please tell me what i just did by doing this, i thought it would create a pull request here directly (first time working with pull requests)

File: README.MD [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Seedqueue
+## Description
+SeedQueue is a mod that is meant to replace multi-instancing. Instead of having multiple Minecrafts generating worlds open at the same time, it does it all in just one Minecraft instance. This greatly improves performance, especially for lower end hardware, and is also aimed to make speedrunning more accessible.
+## Latest Release
+Get latest release of seedqueue from [here](https://github.com/KingContaria/seedqueue/releases).
```
Comment (KingContaria) [inline README.MD:5]:
```suggestion
Get the latest release of SeedQueue from [here](https://github.com/KingContaria/seedqueue/releases).
```

File: README.MD [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+## Tutorial to get started with Seedqueue
+If you are new and want to get started with seedqueue watch this [tutorial](https://www.youtube.com/watch?v=fGu2MYZxh_c).
```
Comment (KingContaria) [inline README.MD:7]: Change: Please replace all occurances of "seedqueue" with "SeedQueue".

File: README.MD [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+## Seedqueue Wiki page
+You can see the seedqueue wiki [here](https://github.com/KingContaria/seedqueue/wiki).
```
Comment (KingContaria) [inline README.MD:9]: Request: Please elaborate on what the wiki explains.

File: README.MD [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Seedqueue
+## Description
+SeedQueue is a mod that is meant to replace multi-instancing. Instead of having multiple Minecrafts generating worlds open at the same time, it does it all in just one Minecraft instance. This greatly improves performance, especially for lower end hardware, and is also aimed to make speedrunning more accessible.
```
Comment (KingContaria) [inline README.MD:3]: Request: Please add information about the wall screen, you can take inspiration from the SeedQueue 1.0 release message.

**Review by KingContaria — CHANGES_REQUESTED** [review]
Is it possible to remove the green lines on the left from the non-titles? I think you might have to add spaces between titles and bodies of text.

**Comment by KingContaria** [comment]
> i have no idea what [Merge branch 'KingContaria:main' into Crystal15118-pull-requests](https://github.com/KingContaria/seedqueue/pull/28/commits/3278e1cade2ccda8388efe1e5613d338093c6a77) did so if anyone knows please tell me what i just did by doing this, i thought it would create a pull request here directly (first time working with pull requests, sorry)

It just merged my recent update into your branch :)

**Comment by KingContaria** [comment]
Fixes https://github.com/KingContaria/seedqueue/issues/25

File: README.MD (hunk 4 above)
Comment (Crystal15118) [reply README.MD:3]: > Request: Please add information about the wall screen, you can take inspiration from the SeedQueue 1.0 release message.

@KingContaria i'll get back to this readme in at night, right now i have some work

**Comment by memerson12** [comment]
Should there be a mention of SpeedAPI and Atum as dependencies in the readme?

**Comment by tildejustin** [comment]
as they are dependencies, I would say yes (I think contaria is asleep)

**Comment by Crystal15118** [comment]
> Is it possible to remove the green lines on the left from the non-titles? I think you might have to add spaces between titles and bodies of text.

not sure what you meant here, can you elaborate more please?

**Review by KingContaria — APPROVED** [review]
Lgtm, thank you! :) I won't close https://github.com/KingContaria/seedqueue/issues/25 with this PR, that will be done when a more in-depth README is written, see https://github.com/KingContaria/seedqueue/pull/31.

**Pull request merged**